### ISSUE TITLE
globalThis causes compile errors

### ISSUE THREAD

**Issue by jonaskello** [stephenh/ts-proto#191]
I upgraded from an earlier version and it seems now there is extra code generated for `globalThis`. This causes a compile error for me as shown below:

```
 error TS7022: 'globalThis' implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer.

8 var globalThis = (() => {
      ~~~~~~~~~~
```

I guess my tsconfig options are too strict to allow this code.

Now it is generated as 

```ts
declare var self: any | undefined;
declare var window: any | undefined;
var globalThis = (() => {
  if (typeof globalThis !== "undefined") return globalThis;
  if (typeof self !== "undefined") return self;
  if (typeof window !== "undefined") return window;
  if (typeof global !== "undefined") return global;
  throw new Error("Unable to locate global object");
})();
```

If I add `any` for the `globalThis` variable like this it compiles:

```ts
declare var self: any | undefined;
declare var window: any | undefined;
var globalThis: any = (() => {
  if (typeof globalThis !== "undefined") return globalThis;
  if (typeof self !== "undefined") return self;
  if (typeof window !== "undefined") return window;
  if (typeof global !== "undefined") return global;
  throw new Error("Unable to locate global object");
})();
```

Could this `any` declaration for `globalThis` be added by default?

**Comment by stephenh** [comment]
@jonaskello I unwittingly fixed this in v1.48.0, can you try upgrading and confirm? Thanks!

**Comment by jonaskello** [comment]
Yes, I upgraded to v1.49.0 now and the `globalThis` code block is no longer generated so that fixes it. Thanks for fixing this :-).
